app/styles/components/az-bars.scss

```diff
@@ -0,0 +1,19 @@
+az-bars {
+  width: calc(100% - 96px);
+  div {
+    text-align: right;
+    padding: 5px;
+    height: 30px;
+    span {
+      width: 100%;
+      padding-right: 5px;
+      display: inline-block;
+      &.Creature {background-color: map-get($color-red, '300');}
```

Comment:
The bar colors here could possibly be associated with card colors at first glance, but I do like the splash of color.